chapters/03_feed-forward.tex

```diff
@@ -29,9 +29,13 @@ \chapter{Feed-Forward Layers}
 \begin{definition}{GELU}{}
     Gaussian Error Linear Unit (GELU) is a non-linear activation function. $\GELU : \R^d\to \R^d$ is computed as follows, where $\Phi$ is the cumulative distribution function of the standard normal distribution and $\sigma(x) = 1/(1+e^{-x})$ \cite{hendrycks2023}.
 
-    \[\GELU(x) = x\,\Phi(x) \approx \sigma(1.702 x).\]
+    \begin{align}
+        \GELU(x) &= x\,\Phi(x) \\
+        &\approx \sigma(1.702 x)\\
```

Comment:
It would be nice (though not our job) to explain how these three expressions are related

Reply:
I agree. In this regard, I've added an intermediary GELU definition.